files/blocks/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "blocks",
+  "repo": "astoilkov/jsblocks",
+  "files": {
+    "basePath": "dist/"
+  }
```

Comment:
Is this going to include all files in all directories. Including the .js and .map files in `mvc` and `query` folders?